test/parse/module/bad-memory-shared-nomax.txt

```diff
@@ -0,0 +1,6 @@
+;;; ERROR: 1
+(module (memory 1 shared))
+
+(;; STDERR ;;;
+Halp I'm getting the wrong error!
```

Comment:
I'm getting some kind of weird error inside PeekMatch here (i.e. we are PeekMatch'ing on Nat and reading "shared").

Reply:
Hm, weird. Maybe will go away with text format from spec...? :-)

Reply:
\o/